src/selectable-group.js

```diff
@@ -65,16 +72,25 @@ class SelectableGroup extends React.Component {
 	 * Called while moving the mouse with the button down. Changes the boundaries
 	 * of the selection box
 	 */
-	_openSelector (e) {		
+	_openSelector (e) {	
+		if(this.state.mouseMoveStarted) return;
+		this.state.mouseMoveStarted = true;
```

Comment:
This isn't the correct way to mutate state. 

Does this really need to be in state? If it doesn't affect the UI, then probably not.